File: src/main/java/br/com/blz/testjava/dto/WarehouseDto.java
package br.com.blz.testjava.dto;

import java.io.Serializable;

public class WarehouseDto implements Serializable {

	private static final long serialVersionUID = 1L;

	private String locality;

	private int quantity;

	private String type;

	public WarehouseDto() {
	}

	public WarehouseDto(String locality, int quantity, String type) {
		this.locality = locality;
		this.quantity = quantity;
		this.type = type;
	}

	public String getLocality() {
		return locality;
	}

	public void setLocality(String locality) {
		this.locality = locality;
	}

	public int getQuantity() {
		return quantity;
	}

	public void setQuantity(int quantity) {
		this.quantity = quantity;
	}

	public String getType() {
		return type;
	}

	public void setType(String type) {
		this.type = type;
	}

}

File: src/main/kotlin/br/com/blz/testjava/dto/WareHouseDto.kt
package br.com.blz.testjava.dto

import br.com.blz.testjava.shared.enum.WarehouseType
import com.fasterxml.jackson.annotation.JsonProperty

class WareHouseDto (

    @JsonProperty("locality")
    var locality: String = "",

    @JsonProperty("quantity")
    var quantity: Int = 0,

    @JsonProperty("type")
    var type: WarehouseType? = null

)
